### DISCUSSION TITLE
Equipo 2 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#9] [Proyecto Final]
Hola @LaluFS y Estefania
Aquí está el hilo en el foro para su equipo. Podrían conversar aquí sobre sus avances con el proyecto y avisar el Equipo de Educación de Patrones Hermosos sobre cualquier pregunta que tengan al respecto.
Con la emoción de seguir el desarrollo de su proyecto, 
Kasturi

**Comment by KasturiSShah** [comment]
Hola! He enviado un correo a ustedes con seguimiento sobre la etapa 0 del proyecto.

**Comment by LaluFS** [comment]
Buen día Kasturi, 

Hemos ido avanzando con las encuestas de nuestro trabajo: https://docs.google.com/document/d/12GaUmz-cbQEIKhpge6jkC77lBtt6h1D9ClWtHduRydE/edit y además hemos ordenado en el jamboard la síntesis de los apuntes https://jamboard.google.com/d/1ZjI7zWSj7Jg4wKHRxxFiwVj9JrOFgh1fLbhc7o-LGa0/viewer?f=0

**Reply by KasturiSShah** [reply]
Gracias por la actualización @LaluFS 
Cuando esté listo la entrega, me podrías avisar? Veo que han completado algunas entrevistas (¡wuhu!), pero hay que rellenar el archivo para la etapa 0 también. 
Me avisan cualquier cosa 😄

**Reply by KasturiSShah** [reply]
Hola @LaluFS y Estefania,

Espero estén muy bien. Gracias por entregar la etapa 0. Veo que han completado las entrevistas todavía, entonces en vez de compartir retroalimentación sobre las preguntas, las podría ofrecer discusión por aquí sobre la(s) mapa(s) que crearían para organizar sus apuntes. ¿Cómo andan con las mapas?

Según la charla con @LaluFS ayer en la asesoría, estoy de acuerdo que crear una mapa por pregunta generaría muchas mapas. Una posibilidad sería crear una mapa de empatía por usuario, luego identificar apuntes comunes entre las mapas, y usar estos puntos comunes para los subtítulos dentro de una mapa de afinidad. O simplemente para después de haber creado mapas de empatía.

Me avisan cualquier cosa, y feliz día internacional de la mujer,
Kasturi

**Comment by LaluFS** [comment]
Hola Kasturi,
Aún no te hemos notificados porque queremos hacer unos cambios en los mapas, si tenemos en el joamboard los mapas de afinidad pero quería modificar alguno para también tener un mapa de empatía al menos: https://jamboard.google.com/d/1ZjI7zWSj7Jg4wKHRxxFiwVj9JrOFgh1fLbhc7o-LGa0/viewer te dejamos la liga para que puedas ir viendo los mapas de afinidad que habíamos avanzado y que aún modificaremos probablemente mañana para que pueda ser revisado máximo el viernes. Agradecemos tus recomendaciones y procederemos a culminar el trabajo.
Que pases un feliz día de la mujer!  <3

**Reply by KasturiSShah** [reply]
Hola @LaluFS 
Gracias por avisar, estaré muy pendiente a este hilo. Cuando me digan que esté listo para mi revisión, entraré para insertar comentarios. La fecha límite es este domingo para la etapa 1 (mapas + planteamiento del problema), entonces no hay prisa.
Para aclarar, me parece que las mapas que crearon para cada pregunta son muy útiles, y no creo que sería necesario borrarlas (simplemente generar algunas mapas más, como mapa de empatía o mapa de afinidad). Por ejemplo, la mapa en la filmina 5 sobre cursos es muy útil, es cómo una lluvia de ideas, y podría informar a su ideación. 
Veo claramente que están poniendo mucho trabajo y pensamiento en este proyecto, gracias, y creo que el proyecto reflejará todo este esfuerzo 😄 
Kasturi

**Reply by LaluFS** [reply]
@KasturiSShah ya está puesto en la carpeta el resumen de la etapa 1 de nuestro proyecto

**Reply by KasturiSShah** [reply]
Hola @LaluFS 
Gracias por la actualización. Revisaré la etapa 1 pronto, y escribiré a ustedes con retroalimentación por este medio.
Saludos,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @LaluFS y Estefania

Gracias por entregar la etapa 1 del proyecto final. Ví no solo el formulario en Github pero también el Jamboard dónde han creado muchas mapas (¡veo 5 mapas de empatía por cada entrevista!). Quisiera felicitarlas por la cantidad de trabajo que han hecho: las entrevistas y también las mapas. 

Es interesante que la mapa de "afinidad" agrupada por pregunta destaca las perspectivas diferentes en las entrevistas, y las mapas de empatía destacan las diferencias entre los usuarios. Mientras que hacen la ideación esta semana, mi sugerencia sería basar sus ideas en las entrevistas, tomando en mente las perspectivas diferentes (en la mapa de afinidad) y qué relación tiene estas perspectivas con los usuarios (en las mapas de empatía).

Para la etapa 2, una sugerencia de mi parte sería hacer la actividad 8S (Ocho Loco) individualmente y luego usar SCAMPER (u otra estrategias en los videos) para ordenar las ideas y elegir una sola idea con que trabajarán en la etapa 3. (Es decir, repetir el proceso que practicamos durante las sesiones interactivas esta semana.)

¿Tienen preguntas para mí? Cualquier cosa, aquí ando.

Mucha suerte,
Kasturi

**Reply by LaluFS** [reply]
Hola Kasturi, 

Hemos avanzado este prototipo low, a ver si nos puedes dar feedback. Esta en miro: https://miro.com/app/board/uXjVMZB13qc=/?share_link_id=20741690697
Además Estefanía y yo estábamos pensando trabajar una versión high para presentar como trabajo final, quedamos atentas a tus comentarios de antemano gracias.

![image](https://user-images.githubusercontent.com/100322488/227797680-c6644b53-d8d4-4589-bb06-ce8aa920a14c.png)

**Reply by KasturiSShah** [reply]
Hola @LaluFS y Estefania
Gracias por mandarme el prototipo low. Para aclarar, ¿están pensando en un prototipo high para su entrega final del proyecto (es decir, la etapa 6)?
Quedo atenta para poder revisar cuando entiendo el objetivo 😄 
Kasturi

**Comment by KasturiSShah** [comment]
Hola @LaluFS y Estefanía,

Gracias por entregar las etapas 2 y 3. He revisado ambas etapas, los vídeos, capturas de pantalla, y el Miro. En primer lugar, felicidades por el montón de trabajo que han hecho. Veo claramente que están poniendo mucho pensamiento y esfuerzo en crear sta iniciativa educativa  para jóvenes. Escribo con retroalimentación. 

Tengo un par de sugerencias:

- Cómo mencionaron en SCAMPER de la etapa 2, creo que lo importante sería unir todo (el sitio web con cursos, el app, Instagram, redes sociales) bajo de un solo "brand" para crear una historia concisa y completa de la iniciativa. Al mirar el Miro y otras diagramas, no tengo muy claro cómo crear el branding. Es muy importante, y el alcance puede ser complicado sin ello.
- Algunas sugerencias al respecto: al principio, es necesario tener app y sitio web? ¿Podrían elegir una, por ejemplo, el sitio web y poner toda la información/cursos etc ahí? De esa manera, podrían crear iconos a las redes sociales (Instagram etc) en el sito web.
- Para el sitio web, bajo de "Contacto", pueden añadir redes sociales ahí.
- En prototipado, desde lo que entiendo, la diagrama de flujo es un prototipado low fidelity, ¿es así? Y la captura de pantalla del sitio web es high fidelity? Y, ¿cómo crearon el sitio web en el vídeo de la etapa 3? 
- El alcance a jóvenes es muy difícil por las razones que mencionó Estefanía en el vídeo. Acá sería el reto muy grande ... por ejemplo, alcanzar a ellos directamente en redes sociales o indirectamente por los padres, o escualas, o gobiernos provinciales, etc. **Mi sugerencia aquí sería definir tu "audiencia", es decir, quién estarán usando _Educación Hoy_, y luego crear un plan de alcance.**

Mi sugerencia más concisa sería enfocarse en uno o dos aspectos de esa iniciativa. Puede ser el alcance, lo veo como uno de los grandes retos. Para poder probar, testear y implementar cambios en los demás módulos.

Finalmente, ¿sería posible discutir su idea con ustedes, quizás en una asesoría? Me encantaría hacerlo. Veo muchos alineamientos con la iniciativa Patrones Hermosos y quizás podríamos colaborar/complementar. 

No hay prisa para contestar a este mensaje. Que disfruten de la semana.

¡Felices vacaciones de la Semana Santa!
Kasturi

**Comment by KasturiSShah** [comment]
Hola @EstefaniaVerka @LaluFS, espero todo bien.

Veo el pull request, creo que @LaluFS ha actualizado su etapa 4. ¿Están suficientes sus cambios, o quieres que acepto tus cambios (reemplazará los cambios de Lau con los tuyos). Quedo al pendiente.

Estefy, creo que no puedes hacer cambios en este repositorio porque tu invitación al repositorio se expiró. Te he re-enviado la invitación. **Por favor, checa tu correo y acéptala para que puedas hacer cambios** 😃 

Cualquier cosa, me avisan.
Kasturi

**Comment by EstefaniaVerka** [comment]
Buen dia Kasturi,

Aaahh ya me parecia.
De por si no me llevo bien con Github y ayer no me dejaba hacer commit y me
enoje jajaj

Gracias por la aclaracion. Con los cambios de Laura estamos ok, yo le pedi
ayuda para subir la etapa 4.

Asi que adelante con los cambios de Lau, quedara asi.

Muchas gracias!

Saludos, Estefi.

El lun, 17 abr 2023 a la(s) 02:46, KasturiSShah ***@***.***)
escribió:

> Hola @EstefaniaVerka <https://github.com/EstefaniaVerka> @LaluFS
> <https://github.com/LaluFS>
>
> Veo el pull request, creo que @LaluFS <https://github.com/LaluFS> ha
> actualizado su etapa 4. ¿Están suficientes sus cambios, o quieres que
> acepto tus cambios (reemplazará los cambios de Lau con los tuyos). Quedo al
> pendiente.
>
> Estefy, creo que no puedes hacer cambios en este repositorio porque tu
> invitación al repositorio se expiró. Te he re-enviado la invitación. *Por
> favor, checa tu correo y acéptala para que puedas hacer cambios* 😃
>
> Cualquier cosa, me avisan.
> Kasturi
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PatronesHermosos/design-thinking-1ra-generacion/discussions/9#discussioncomment-5632518>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYCIWAIHL6DNQ4F6CYHX6I3XBTKL7ANCNFSM6AAAAAAVTB3T5Q>
> .
> You are receiving this because you were mentioned.Message ID:
> <PatronesHermosos/design-thinking-1ra-generacion/repo-discussions/9/comments/5632518
> @github.com>
>


-- 
*Estefania Verka *

**Reply by KasturiSShah** [reply]
Hola @EstefaniaVerka 
Gracias por contestar. Claro, entonces revisaré la etapa 4 con los cambios de Lau.
Kasturi

**Comment by KasturiSShah** [comment]
Hola @LaluFS @EstefaniaVerka 

Gracias por entregar la etapa 4, escribo con retroalimentación. ¡Felicidades por su página web! Quedé sin palabras, es muy desarrollada. Son interesantes los comentarios y sugerencias de los usuarios. Mi recomendación sería mostrar cada parte de la página a varios usuarios (es decir, hacer más pruebas), especialmente porque la página web tiene tantas sub-páginas y tantos elementos que navegar. Así que otra recomendación sería dar una tarea a los usuarios con que charlarán, por ejemplo, "crear una cuenta" o "inicia sesión" o "busca el formulario de microeconomia" o "mira los primeros 3 vídeos del curso inglés". Ellos tendrán que navegar la página web desde la página principal: al mirar su navegación, recibirán mucha información sobre el diseño de la página y cómo hacerlo más navegable. Este tipo de test es muy popular con diseñadores de páginas web/apps, por eso lo recomiendo. No es necesario hacer más pruebas antes de la clausura del curso, solo dejo esta sugerencia para ustedes para el fúturo.

Otras sugerencias que tengo son:
- Me parece que hay muchos aspectos a su proyecto. Una sugerencia sería ir agregando elementos un elemento a la vez, cuando cada elemento esté 100% desarrollado, en vez de tener placeholders para cursos que están bajo desarrollo. De esa manera, crearán una reputación buena de recursos de calidad que están lanzando.
- No tengo muy claro cómo van a desarrollar el contenido para los cursos. O están pensando en buscar vídeos cortos sobre cada curso en la red? Es decir, esta página sería una "biblioteca" para personas interesadas en buscar cursos relevantes?
- Una sugerencia sería agregar una sub-página que explica "Cómo aprenderás". Creo que aclarará bastante cómo un usuario puede interactuar con la página  y que tiene que esperar desde la página web
- ¿Cómo crearon el sitio web? Si usaron una herramienta como Word Press o si usaron una plantilla, es importante dar créditos a estos fuentes (Normalmente, se pone los fuentes en el footer de cada página, por ejemplo, "Creado usando Plantilla XXX de YYY" o "Hecho en WordPress")

Como he mencionado anteriormente, cuando el curso termine, me encantaría platicar más sobre este proyecto con ustedes. Hay muchos alineamientos con Patrones Hermosos, supongo que haya varias ideas / colaboraciones que podemos pensar en hacer.

Finalmente ...
¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar su presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si les conviene la asignación de su equipo a una de las dos sesiones de clausura del curso, o si quieren cambiar su día asignado al otro día.

Estoy aquí para apoyarles, cualquier cosa, aquí estoy!

Mucha suerte, mucho éxito y hasta pronto
Kasturi
